src/library/scala/runtime/VM.java

```diff
@@ -0,0 +1,46 @@
+package scala.runtime;
+
+import java.lang.invoke.MethodHandle;
+import java.lang.invoke.MethodHandles;
+import java.lang.invoke.MethodType;
+import java.lang.reflect.Field;
+
+public class VM {
```

Comment:
sure thing.

Speaking of modifers, I would have preferred to make this package private, and to have indirected call through `ScalaRuntime.releaseFence`. But we don't have a way to force VM inlining of `ScalaRuntime.releaseFence`, so I wanted to keep it inlinable by the Scala inliner.

Reply:
Maybe it would be better, at least initially, to leave inlining of `releaseFence` up to the JDK (my kingdom for `@ForceInline`!), and make `VM` package protected.